canonical-web-and-design/ubuntu.com
# Openstack consulting piechart

## Done

- Create d3 piechart displaying the data from chart https://www.openstack.org/analytics, chart n.197
- Create a table displaying the same results for smaller screens


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/TEMP
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copydoc](https://docs.google.com/document/d/1f83OSIHVQGMjF7b0pZUKFvoubMGqi_b-OYbSpyhveX4/edit#heading=h.gntdvz3k3axy)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4387

## Screenshots

![image](https://user-images.githubusercontent.com/58276363/131700448-96afacde-cf41-444b-833f-ee7673255515.png)
